_drafts/STMViewer.md

```diff
@@ -0,0 +1,161 @@
+---
+# This is a template for Interrupt posts. Use previous, more recent posts from the _posts/
+# directory for more inspiration and patterns.
+#
+# When submitting a post for publishing, submit a PR with the post in the _drafts/ directory
+# and it will be moved to _posts/ on the date of publish.
+#
+# e.g.
+# $ cp _drafts/_template.md _drafts/my_post.md
+#
+# It will now show up in the front page when running Jekyll locally.
+
+title: Visualizing embedded data made easy
```

Comment:
I think the title should probably have a mention of STMViewer. Maybe something like:

```
Visualizing Realtime Data With STMViewer
```